tools/dcsnoop.bt

```diff
@@ -12,6 +12,95 @@
  * Copyright 2018 Netflix, Inc.
  *
  * 08-Sep-2018	Brendan Gregg	Created this.
+ * 
+ * Example of usage:
+ * 
+ * # ./dcsnoop.bt
+ * Attaching 4 probes...
+ * Tracing dcache lookups... Hit Ctrl-C to end.
+ * TIME     PID    COMM             T FILE
+ * 427      1518   irqbalance       R proc/interrupts
```

Comment:
Nit: some of the sample output for these scripts is really long and duplicative. What do you think about truncating some of these?

Reply:
Agreed, 4 lines sounds reasonable.

Reply:
Tried to clean up as much as possible. There are still places where the example usage is long however not duplicative since there is contextual info in the output as described in comments below them.